### DISCUSSION TITLE
missing information under predictive modelling and information services

### DISCUSSION THREAD

**Discussion by AnnetWanyana** [lsc-hubs/kenya-hub#8] [General]
There  is missing information under predictive modelling and information services when you continue to navigate through.

**Comment by AnnetWanyana** [comment]
I noted some data was not filtered under partnerships.

**Comment by pvgenuchten** [comment]
Thank you for the comment @AnnetWanyana, i expect this section is still in development, do you have a suggestion who could contribute this content?

Can you contact me personally about which content has not been filtered yet?
